admin/test/test_installer.py

```diff
@@ -0,0 +1,103 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+"""
+Tests for ``admin.installer``.
+"""
+from textwrap import dedent
+
+from pyrsistent import PClass, field, pmap_field
+
+from flocker.testtools import (
+    TestCase
+)
+
+from ..installer._images import (
+    _packer_amis, _PackerOutputParser,
+)
+
+
+class ParserData(PClass):
```

Comment:
:grey_question: I don't think it adds much, but some people would want a docstring here that describes ParserData and its instance variables.

In this case that seems like more of a maintenance burden.

I'm only commenting just in case you meant to add a docstring and just forgot. Otherwise ignore this comment and move on.